src/components/Blogs/Blogs.jsx

```diff
@@ -21,22 +31,40 @@ const Blogs = () => {
         (state) => state.userDetail,
     );
 
+    const followUserId = user?._id;
```

Comment:
This name is misleading, this is not a user id of who we follow . It is our user id. Which used to get info about users that we follow after.

Reply:
followUserId is changed to userId